app/views/candidate_interface/sign_up/new.html.erb

```diff
@@ -3,17 +3,33 @@
 
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-xl">
-      <%= t('authentication.sign_up.heading') %>
-    </h1>
 
-    <p class="govuk-body-l">Please give us your email address so that you can save and return to your application.</p>
-
-    <%= form_with model: @candidate, url: candidate_interface_sign_up_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: true } do |f| %>
+    <%= form_with model: @sign_up_form, url: candidate_interface_sign_up_path, builder: GOVUKDesignSystemFormBuilder::FormBuilder, html: { novalidate: true } do |f| %>
       <%= f.govuk_error_summary %>
 
+      <h1 class="govuk-heading-xl">
+        <%= t('authentication.sign_up.heading') %>
+      </h1>
+
       <%= f.govuk_text_field :email_address, label: { text: t('authentication.sign_up.email_address.label'), size: 'm' }, hint_text: t('authentication.sign_up.email_address.hint_label'), type: :email, width: 'two-thirds' %>
 
+      <h2 class="govuk-heading-m">You won’t need a password to use this service</h2>
+      <p class="govuk-body">Instead, you’ll sign in using your email address. Each time you sign in, we'll send you a link so you can return to your application.</p>
+
+      <h2 class="govuk-heading-m">Terms and conditions</h2>
+      <p class="govuk-body">Our <a href="/terms-conditions">terms and conditions</a> contain important information about:</p>
```

Comment:
We have a view helper called `govuk_link_to` that can be used here which will add in the `govuk-link` CSS class.

E.g. `<%= govuk_link_to 'terms and conditions', '/terms-conditions' %>`